runtime/src/bank.rs

```diff
@@ -10646,6 +10661,53 @@ pub(crate) mod tests {
         assert_eq!(bank.get_balance(&pubkey), amount);
     }
 
+    #[test]
+    fn test_executed_transaction_count_pre_bank_transaction_count_fix() {
```

Comment:
Sorry, one more request. In both of these tests, could you add a child Bank that processes more transactions to verify the accumulation vs. single-bank behavior? I think that will be nice to have when we add a persistent executed_transaction_count field to Bank/snapshots.

Reply:
No problem. Done